website/contributing/how-to-build-from-source.md

```diff
@@ -12,8 +12,8 @@ Assuming you have the Android SDK installed, run `android` to open the Android S
 
 Make sure you have the following installed:
 
-1. Android SDK version 30 (`compileSdkVersion` in [`build.gradle`](https://github.com/facebook/react-native/blob/main/ReactAndroid/build.gradle)).
-2. Android NDK 21.4.7075529 (from in [`gradle.properties`](https://github.com/facebook/react-native/blob/b1120c6a6513ec24568abc6bbe60dd57ef1dec96/gradle.properties#L7), download links and installation instructions below).
+1. Android SDK (find the specific version in the `compileSdkVersion` in [`build.gradle`](https://github.com/facebook/react-native/blob/main/ReactAndroid/build.gradle#L253)).
```

Comment:
Should we use a permalink? This Line reference will keep on breaking in time

Reply:
which is why I added the specification of `compileSdkVersion` - my concern with a permalink is that folks will see an outdated version of the file and not realize it. I rather point always to the file on `main` and then have folks look for `compileSdkVersion` instead of risking them seeing an outdated version